raiden/api/mock.py

```diff
@@ -33,7 +33,7 @@ class MockAPI(object):
     """
     channel_by_address = dict() # 1:1
     all_channel = [] # N
-    channels_by_asset = defaultdict(list) # 1:N
+    channels_by_token = defaultdict(list) # 1:N
     channels_by_partner = defaultdict(list) # 1:N
```

Comment:
Will remove this file probably in the next PR in case I still need it for manual experimentation.